### DISCUSSION TITLE
18 gennaio 2022 A - esercizio 3

### DISCUSSION THREAD

**Discussion by IonutCicio** [sapienzastudentsnetwork/basi-di-dati-1#4] [Esami]
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/40034429/cae2e483-07ed-4fb7-a8fd-c25ffbd9f286)

**Comment by IonutCicio** [comment]
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/40034429/9e377ce6-3ca6-4074-8aa1-859edf4a3d2d)

**Comment by Frisbon** [comment]
Analogo a sopra (vero?)

![image_2024-01-14_10-38-51](https://github.com/sapienzastudentsnetwork/bdi2324/assets/84731341/324145cf-72b9-481c-a054-81ed15e9d816)

**Comment by niccolo-fato** [comment]
![IMAGE 2024-01-14 11:01:06](https://github.com/sapienzastudentsnetwork/bdi2324/assets/73995334/b3e0fbf6-1f34-4419-aea1-57bd8ef0e47e)

**Comment by daromacarlo** [comment]
![Immagine 2024-02-05 170541](https://github.com/sapienzastudentsnetwork/bdi2324/assets/135441131/88909448-10d2-4190-90d9-fabd8fb74a9d)
![Immagine 2024-02-05 170553](https://github.com/sapienzastudentsnetwork/bdi2324/assets/135441131/5ccbaf79-eace-43f4-8a7b-76dacc410d41)
![Immagine 2024-02-05 170602](https://github.com/sapienzastudentsnetwork/bdi2324/assets/135441131/64b5baaa-5206-47b4-9304-cd71e27a4934)

**Reply by Frisbon** [reply]
I Bucket vuoti su Minecraft sono stack-abili fino a 16, quindi sono >42 stack

**Reply by daromacarlo** [reply]
@Frisbon tu sei uno che ne sa! non me la ricordavo sta cosa

**Comment by M1SC0** [comment]
<img width="1669" height="762" alt="image" src="https://github.com/user-attachments/assets/2f8de8ab-ab84-452e-bf0a-af10a848c87a" />
